tests/hardware_tests/test_nav_env.py

```diff
@@ -41,20 +47,29 @@ def init_config():
 
 
 def compute_avg_and_std_time(ref_traj_set):
+    """
+    Compute average and standard deviation of time taken to reach each waypoint for all trajectories in ref_traj_set
+
+    Args:
+        ref_traj_set: List(List(List(Dict))) where each Dict is a waypoint
```

Comment:
what are the different dimensions of this list?

Reply:
I prefer just saying list of recorded trajectories :)

Reply:
Dict = waypoint (timestamp, pose)
List(Dict) = trajectory for each goal waypoint
List(List(Dict)) = trajectory list for all "n" waypoints robot was asked to go to
List(List(List(Dict))) = aggregation of 15 reference trajectories of type List(List(Dict)) as a List

Reply:
added